runner/gateway.openapi.yaml

```diff
@@ -878,11 +876,9 @@ components:
           title: Finish Reason
           default: ''
         delta:
-          allOf:
-          - $ref: '#/components/schemas/LLMMessage'
+          $ref: '#/components/schemas/LLMMessage'
         message:
-          allOf:
-          - $ref: '#/components/schemas/LLMMessage'
+          $ref: '#/components/schemas/LLMMessage'
```

Comment:
I think you need to revert this OpenAPI changes from `allOf`, because the CI is failing. Maybe you're using some different lib version for generating OpenAPI.

Reply:
Ah yeah this is an [older syntax](https://stackoverflow.com/a/73490368). Just tried a bunch of things to get CI to pick up the newer syntax (updated fastapi, updated python) but nothing seemed to work so I just manually picked the changes in https://github.com/livepeer/ai-runner/pull/579/commits/0aaed81f4ade646083dd2ef491b028c36357fd9a

> In OpenAPI 3.1 (which uses JSON Schema 2020-12 by default), there's even no need for allOf if you only have one $ref because $ref now [allows sibling keywords](https://stackoverflow.com/a/41752575/113116).